nims-notes-things/src/components/Form.js

```diff
@@ -12,19 +12,17 @@ const Form = (props) => {
     name: "",
     date: "",
     note: "",
-  });
-
-  const [empty, setFormEmpty] = useState({
-    name: false,
-    date: false,
-    note: false,
+    valid: false,
```

Comment:
Is this needed? haven’t seen any use of this property

Reply:
No it isn't needed - i forgot to take it out.